cabal-install/cabal-install.cabal

```diff
@@ -12,68 +9,46 @@ Description:
     and installation of Haskell libraries and programs.
 homepage:           http://www.haskell.org/cabal/
 bug-reports:        https://github.com/haskell/cabal/issues
-License:            BSD3
+License:            BSD-3-Clause
 License-File:       LICENSE
 Author:             Cabal Development Team (see AUTHORS file)
 Maintainer:         Cabal Development Team <cabal-devel@haskell.org>
 Copyright:          2003-2020, Cabal Development Team
 Category:           Distribution
 Build-type:         Simple
 Extra-Source-Files:
-  README.md bash-completion/cabal changelog
+  README.md
+  bash-completion/cabal
+  changelog
 
 source-repository head
   type:     git
   location: https://github.com/haskell/cabal/
   subdir:   cabal-install
 
 Flag native-dns
-  description:  Enable use of the [resolv](https://hackage.haskell.org/package/resolv) & [windns](https://hackage.haskell.org/package/windns) packages for performing DNS lookups
+  description:
+    Enable use of the [resolv](https://hackage.haskell.org/package/resolv)
+    & [windns](https://hackage.haskell.org/package/windns) packages for performing DNS lookups
   default:      True
   manual:       True
 
-Flag debug-expensive-assertions
-  description:  Enable expensive assertions for testing or debugging
-  default:      False
-  manual:       True
-
-Flag debug-conflict-sets
-  description:  Add additional information to ConflictSets
-  default:      False
-  manual:       True
-
-Flag debug-tracetree
-  description:  Compile in support for tracetree (used to debug the solver)
-  default:      False
-  manual:       True
-
 Flag lukko
   description:  Use @lukko@ for file-locking
   default:      True
   manual:       True
 
-executable cabal
-    main-is:        Main.hs
-    hs-source-dirs: main
-    default-language: Haskell2010
-    ghc-options:    -Wall -fwarn-tabs -fwarn-incomplete-uni-patterns -fwarn-incomplete-record-updates
-    if impl(ghc >= 8.0)
-        ghc-options: -Wcompat
-                     -Wnoncanonical-monad-instances
-      if impl(ghc < 8.8)
-        ghc-options: -Wnoncanonical-monadfail-instances
 
+library
+    default-language: Haskell2010
+    ghc-options: -Wall -Wcompat -Wnoncanonical-monad-instances
+    if impl(ghc < 8.8)
+      ghc-options: -Wnoncanonical-monadfail-instances
       if impl(ghc >=8.10)
         ghc-options: -Wunused-packages
```

Comment:
What about using common stanzas now that we can? So that we only need to update this stuff in one place

Reply:
Oo! good point. In the previous iteration i had `cabal-version: 2.0`, but `2.2` seemed like a better idea for internal libs. We definitely can do common stanzas.

Reply:
Done